src/glyph/serialize.rs

```diff
@@ -111,9 +111,25 @@ impl Glyph {
         }
 
         writer.write_event(Event::End(BytesEnd::borrowed(b"glyph")))?;
-        writer.inner().write_all("\n".as_bytes())?;
-        writer.inner().flush()?;
 
+        // Write `\r\n` final line ending on Win by default
+        // Write `\n` final line ending on all other platforms by default
+        if cfg!(windows) && !options.line_ending_normalization {
```

Comment:
This is a change in the current default approach to the final line ending in glif files on all platforms.  This will now write a final `\r\n` on Win and `\n` on non-Win.  This is consistent with the decision about how to approach defaults and customization support.